README.md

```diff
@@ -1,2 +1,41 @@
 # scala-native-cli
-Command Line Interface for Scala Native
+
+Command Line Interface for the tools component of Scala Native. Tools handle the linking, optimisation and code generation phases. 
+This means that the initial .scala files cannot be compiled with it - for that look to the nscplugin component. 
+
+Please note that this api is meant for Scala Native power users. For a more friendly Scala Native building experience please use 
+the dedicated [sbt plugin](https://scala-native.readthedocs.io/en/stable/user/sbt.html#minimal-sbt-project) or [scala-cli](https://scala-cli.virtuslab.org).
+
+## Usage
+
+scala-native-cli --help will bring up possible options and syntax. 
+Since options are dependent on the used version of scala-native-cli, please use it for further instructions.
+
+### Passing link-time resolved properties
```

Comment:
I had no idea before how many problems expecting to pass `Any` in link-time properties would make. I believe we should change the logic in Scala Native to use passed value, but if the type does not match it should try to parse it. Luckily until 0.4.2 release all link-time properties are defined only as booleans (isWindows, isWeakRefSupported) and users cannot define new ones. 
For 0.4.x releases we should only parse strings to booleans, and fail if some other type would be passed. For 0.5.x parsing would be done by Scala Native, we would pass only raw strings.